packages/shared_preferences/shared_preferences_platform_interface/lib/method_channel_shared_preferences.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+import 'shared_preferences_platform_interface.dart';
+
+const MethodChannel _kChannel =
+    MethodChannel('plugins.flutter.io/shared_preferences');
+
+/// Wraps NSUserDefaults (on iOS) and SharedPreferences (on Android), providing
+/// a persistent store for simple data.
+///
+/// Data is persisted to disk asynchronously.
+class MethodChannelSharedPreferencesStore
```

Comment:
Did you just refactor this file and made it only implement the things that are .invoked in the old plugin?

Reply:
Indeed he did :P https://github.com/flutter/plugins/pull/2291#issuecomment-557754517